src/Husky/Husky.csproj

```diff
@@ -49,15 +49,9 @@
     <PackageReference Include="Microsoft.Extensions.FileSystemGlobbing" Version="6.0.0" />
     <PackageReference Include="System.IO.Abstractions" Version="17.0.13" />
   </ItemGroup>
-  <ItemGroup>
-    <Compile Remove="Utils\Dotnet\ParallelRangeManager.cs" />
-    <Compile Remove="Utils\Dotnet\ParallelLoopState.cs" />
-    <Compile Remove="Utils\Dotnet\ParallelForEach.cs" />
-    <Compile Remove="Utils\Dotnet\ParallelETWProvider.cs" />
-    <Compile Remove="Utils\Dotnet\Parallel.cs" />
-  </ItemGroup>
-  <Target Name="Husky" BeforeTargets="Restore;CollectPackageReferences" Condition="'$(HUSKY)' != 0 and '$(IsCrossTargetingBuild)' == 'true'">
-    <Exec Command="dotnet tool restore" StandardOutputImportance="Low" StandardErrorImportance="High" />
-    <Exec Command="dotnet husky install" StandardOutputImportance="Low" StandardErrorImportance="High" WorkingDirectory="..\.." />
-  </Target>
+    <!--  TODO: re-enable once husky supports .NET 7-->
```

Comment:
It doesn't build on my machine, I get a error message that I'm missing .NET 6.0 since the published version of Husky doesn't specify .NET 7.0 as a target.

Reply:
My bad, I figured out how to install multiple SDKs together, should be good now. 😄